WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -100,6 +100,9 @@ def wca_table(responsive: true, hover: true, striped: true, floatThead: true, ta
     if striped
       table_classes += " table-striped"
     end
+    if greedy
```

Comment:
It doesn't look ugly, but when filtering client size the columns are resized (because the "name" and "tags" column resizes to fit the longest incident name/tags).
I didn't like this behavior and found it annoying, so I fixed the column sizes by removing the greedy column.